.github/workflows/catnapw.yml

```diff
@@ -18,204 +18,97 @@ on:
       - master
   workflow_dispatch:
 
-jobs:
-
-  #=====================================================================================================================
-  # Setup
-  #=====================================================================================================================
-
-  # Node 0
-  setup-demikernel0:
-    name: Setup Node 0
-    runs-on: ubuntu-latest
-    steps:
-    - name: Fetch Changes
-      uses: appleboy/ssh-action@master
-      with:
-        host: ${{ secrets.CATNAPW_HOSTNAME_A }}
-        port: ${{ secrets.PORTNUM }}
-        key: ${{ secrets.SSHKEY }}
-        username: ${{ secrets.USERNAME }}
-        script: |
-          cd c:\demikernel
-          $GitPath = Join-Path $Env:ProgramFiles "\Git\cmd"
-          $Env:Path += $GitPath + ';'
-          git pull
-          git checkout origin/${{ github.ref_name }}
-
-
-  # Node 1
-  setup-demikernel1:
-    name: Setup Node 1
-    runs-on: ubuntu-latest
-    steps:
-    - name: Fetch Changes
-      uses: appleboy/ssh-action@master
-      with:
-        host: ${{ secrets.CATNAPW_HOSTNAME_B }}
-        port: ${{ secrets.PORTNUM }}
-        key: ${{ secrets.SSHKEY }}
-        username: ${{ secrets.USERNAME }}
-        script: |
-          cd c:\demikernel
-          $GitPath = Join-Path $Env:ProgramFiles "\Git\cmd"
-          $Env:Path += $GitPath + ';'
-          git pull
-          git checkout origin/${{ github.ref_name }}
-
-  #=====================================================================================================================
-  # Setup Completed
-  #=====================================================================================================================
-
-  setup:
-    name: Setup Completed
-    needs: [setup-demikernel0, setup-demikernel1]
-    runs-on: ubuntu-latest
-    steps:
-      - name: Log
-        run: echo "Setup Completed!"
-
-  #=====================================================================================================================
-  # Build
-  #=====================================================================================================================
+env:
+  LIBOS: catnapw
```

Comment:
Do we still need to use `catnapw`? We have already changed the rust code and the Makefiles to `catnap`.

Reply:
Yes, we do in `demikernel_ci.py`. I intend to address this in future PRs, because I understand that integrating Windows ASAP is more important than addressing naming schemes.